### ISSUE TITLE
Inclusion of rebar3_scuttler breaks args to rebar3 release

### ISSUE THREAD

**Issue by jozuas** [lrascao/rebar3_scuttler#4]
Hi!

I noticed today that as soon as `rebar3_scuttler` is included to `project_plugins`, `rebar3 release --all` or `rebar3 release --relname some_name` results in "Invalid option --relname on task release".

Minimal example for reproducing:

```bash
git clone https://github.com/galdor/rebar3-test.git
cd rebar3-test
rebar3 release --relname app1
# Success
```

Then, add to `rebar.config`

```erlang
{project_plugins, [
        {rebar3_scuttler, {git, "https://github.com/lrascao/rebar3_scuttler", {tag, "0.5.0"}}}
]}.
```

Test

```bash
rebar3 release --relname app1
# ===> Invalid option --relname on task release
```

**Comment by lrascao** [comment]
ah, i think it's because scuttler stubs out the relx release project in order to be able to inject the cuttlefish calls

**Comment by jozuas** [comment]
Since you've assigned yourself to this ticket, are you planning to work on this? 👼🏻

**Comment by lrascao** [comment]
i'll take all the help i can! Can't really promise when/if i'll be able to pick it up. As a workaround you can try defining the release in `rebar.config` instead of the cli

**Comment by jozuas** [comment]
I've forked your repo and started working on this today. I'll fix the issue and make a PR. This change set is going to be a part of a production system, so the change-set will be tested. I'll let you know if anything changes.

**Comment by lrascao** [comment]
Excellent, thanks!

**Comment by jozuas** [comment]
Hi, just wanted to give a quick heads-up.

This week I had some time to work on the issue. Managed to solve it in a backwards-compatible manner - no changes are needed for users releasing single applications.

Few known rough edges remaining:
- Duplicate files are being included in `rebar3 tar`, all duplicates are bundled in a separate directory.
- `relx` `overlay_vars` per-release configuration is not performing the substitutions correctly. Haven't investigated deeply, but seems like a Relx issue.

 If you are interested or want to give any preliminary feedback, you can find my fork [here](https://github.com/jozuas/rebar3_scuttler).

**Comment by jozuas** [comment]
My team has reflected on `cuttlefish`, `rebar3_scuttler`, and the way that our other projects do configuration. We have decided to stop using cuttlefish across all our Erlang projects.

My fork supports multiple releases from a single codebase with `rebar3_scuttler`. Sadly it is nowhere close to production readiness, and has quite a few issues. I'm uncertain that the way I went about is right. In addition, given my current lack of time, I am not going to be pursuing merging my changes.
